src/commands/add-evm-account.ts

```diff
@@ -0,0 +1,103 @@
+import { Flags } from '@oclif/core'
+import { PinkContractPromise } from '@phala/sdk'
+
+import PhatBaseCommand, { type ParsedFlags, type BrickProfileContract } from '../lib/PhatBaseCommand'
+import { bindWaitPRuntimeFinalized } from '../lib/utils'
+
+export default class AddEvmAccount extends PhatBaseCommand {
+  static description = 'Add EVM accounts'
+
+  static args = {
+    ...PhatBaseCommand.args
+  }
+
+  static flags = {
+    ...PhatBaseCommand.flags,
+    evmRpcEndpoint: Flags.string({
+      description: 'EVM RPC endpoint',
+      required: true,
+    }),
+  }
+
+  public async run(): Promise<void> {
+    const { evmRpcEndpoint } = this.parsedFlags as ParsedFlags & {
+      evmRpcEndpoint: string
+    }
+    const pair = await this.getDecodedPair({
+      suri: this.parsedFlags.suri || process.env.POLKADOT_WALLET_SURI,
+      accountFilePath: this.parsedFlags.accountFilePath || process.env.POLKADOT_WALLET_ACCOUNT_FILE,
+      accountPassword: this.parsedFlags.accountPassword || process.env.POLKADOT_WALLET_ACCOUNT_PASSWORD,
+    })
+
+    // Step 1: Connect to the endpoint.
+    const endpoint = this.getEndpoint()
+    const [apiPromise, registry, cert] = await this.connect({
+      endpoint,
+      pair,
+    })
+
+    // Step 2: Query the brick profile contract id.
+    this.action.start('Querying your Brick Profile contract ID')
+    const brickProfileContractId = await this.getBrickProfileContractId({
+      endpoint,
+      registry,
+      apiPromise,
+      pair,
+      cert,
+    })
+    this.action.succeed(`Your Brick Profile contract ID: ${brickProfileContractId}`)
+
+    // Step 3: generate evm account
+    try {
+      this.action.start('Adding evm account')
+      const brickProfileAbi = await this.loadAbiByContractId(
+        registry,
+        brickProfileContractId
+      )
+      const brickProfileContractKey = await registry.getContractKeyOrFail(
+        brickProfileContractId
+      )
+      const brickProfile: BrickProfileContract = new PinkContractPromise(
+        apiPromise,
+        registry,
+        brickProfileAbi,
+        brickProfileContractId,
+        brickProfileContractKey
+      )
+      const { output } = await brickProfile.query.externalAccountCount(cert.address, {
+        cert,
+      })
+      if (output.isErr) {
+        throw new Error(output.asErr.toString())
+      }
+      const externalAccountCount = output.asOk.toNumber()
+      const waitForPRuntimeFinalized = bindWaitPRuntimeFinalized(registry)
+      await waitForPRuntimeFinalized(
+        brickProfile.send.generateEvmAccount(
+          { cert, address: pair.address, pair },
+          evmRpcEndpoint
+        ),
+        async function () {
+          const { output } = await brickProfile.query.externalAccountCount(cert.address, {
+            cert,
+          })
+          return output.isOk && output.asOk.toNumber() === externalAccountCount + 1
+        }
+      )
```

Comment:
```ts
const result = await brickProfile.send.generateEvmAccount(
  { cert, address: pair.address, pair },
  evmRpcEndpoint
)
await result.waitFinalized(..)
```
check the full example here:
https://github.com/Phala-Network/phat-lego-ui/blob/main/src/features/bricks/BrickProfile.ts#L1211-L1222